packages/cms/custom_migrations/20220825A-migrate-price-currencies.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
+
+module.exports = {
+  async up(knex) {
+    // add default USD, ETH for currencies
+    await knex('currencies').insert({ currency: 'USD' });
+    await knex('currencies').insert({ currency: 'ETH' });
+
+    const priceCurrencies = await knex('price_currencies');
+    const products = await knex('products').whereNotNull('price');
+    const currencies = await knex('currencies');
+
+    const getThingByKey = (things, key, value) => {
+      for (let i = 0; i < things.length; i++) {
+        if (value === things[i][key]) {
+          return things[i];
+        }
+      }
+      return null;
+    };
+
+    for (let i = 0; i < products.length; i++) {
+      const product = products[i];
+      // get price currency for product
+      const priceCurrency = getThingByKey(
+        priceCurrencies,
+        'id',
+        product?.price,
```

Comment:
Instead of optional operator here, should just check if product is nullish above and skip that iteration of the loop (e.g. `if (!product) continue;`)